app/views/appointment_summaries/new.html.erb

```diff
@@ -1,3 +1,4 @@
+<% content_for(:page_title, 'How to view your appointment summary - Pension Wise') %>
```

Comment:
I've used the t('service.title', X) pattern for these missing page titles - gone through hopefully all pages that are missing a title and added one.